apps/ledger-live-mobile/src/components/RootNavigator/MainNavigator.tsx

```diff
@@ -147,6 +148,9 @@ export default function MainNavigator() {
           listeners={({ navigation }) => ({
             tabPress: e => {
               e.preventDefault();
+              track("web3hub_section_clicked", {
+                page: "Home",
+              });
```

Comment:
Again not sure we want to track this as we can just track the screen instead, we never did track this kind of click for other screens and it doesn't tell you if you end up in the actual screen

Reply:
Having TrackScreen on the main page would be called every time he switch from search to main or quiting an app ? If we just wanna track how many timesa user clicl on the Discover then might be better to implement this here